event_sale_extra_info/models/event.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# © 2016 Antiun Ingeniería S.L. - Jairo Llopis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+
+
+class EventRegistration(models.Model):
+    _inherit = "event.registration"
+
+    commercial_partner_id = fields.Many2one(
+        "res.partner",
+        "Commercial partner",
+        related="partner_id.commercial_partner_id",
+        readonly=True,
+        store=True,
+        help="Commercial partner related to the chosen partner.",
+    )
+
+    @api.model
+    def _init_commercial_partner_id(self):
```

Comment:
I think it can be because your test DB, because normally these fields are computed at installation time. Please try on a fresh DB or see on runbot if they are filled.